docs/server/source/server-reference/configuration.md

```diff
@@ -149,6 +156,7 @@ export BIGCHAINDB_SERVER_THREADS=5
 ```js
 "server": {
     "bind": "localhost:9984",
+    "loglevel": "info",
```

Comment:
I just did the following, on the latest master branch:
```text
$ rm ~/.bigchaindb
$ bigchaindb -y configure mongodb
$ cat ~/.bigchaindb
{
    "keypair": {
        "public": "EGG5QvwXZZZJnq3FEgGn8YkW1QaR9FBd8f636ix1yxtj",
        "private": "srLohmpD3jErMRJiQJsKcdsLrtDU1tHb9neVkhtSEL1"
    },
    "backlog_reassign_delay": 120,
    "log": {},
    "database": {
        "max_tries": 3,
        "port": 27017,
        "ssl": false,
        "name": "bigchain",
        "backend": "mongodb",
        "host": "localhost",
        "password": null,
        "replicaset": "bigchain-rs",
        "login": null,
        "connection_timeout": 5000
    },
    "keyring": [],
    "server": {
        "bind": "localhost:9984",
        "workers": null,
        "threads": null
    }
}
```
For some reason, the `server.loglevel` isn't showing up in the default generated config file any more.